### ISSUE TITLE
[Support]: Coral Dev Board 1GB recommended? 

### ISSUE THREAD

**Issue by antonbek89** [blakeblackshear/frigate#2721]
### Describe the problem you are having

This days impossible to get Coral USB

Does anyone here or the developers can give same recommendation about Coral Dev Board 1GB (it's the only one with available suppliers)? 

I want to run Frigate and want its work amazing!!!

### Version

. 

### Frigate config file

```yaml
.
```

### Relevant log output

```shell
....
```

### FFprobe output from your camera

```shell
.
```

### Operating system

Other

### Install method

Docker CLI

### Coral version

Dev Board

### Network connection

Mixed

### Camera make and model

.

**Comment by nickrout** [comment]
I am interested in this answer too.

**Comment by nickrout** [comment]
Thanks, I found an online retailer with a NZ branch (RS online) who say they expect a usb coral on 25 February, so fingers crossed.

**Comment by gszakacs** [comment]
I was in the same shoes, but I did not want to wait so I ended up buying the 1GB dev board for ~$60. Understanding the limitations (coral dev board has no HW encoder, limited ram/storage), I decided to use it only for object detection/tracking and taking snaphots for which it has been working great ! A miniPC is running separately to run a docker serving as an rtsp bridge for some cheap wifi color night vision cameras (wyze v3) and saving the HD streams 24/7. This way, I get ~ 15ms inference time for 4 cameras and the coral dev board CPU utilization is < 50% (max permissible utilization is 400% because of the quad core cpu of the coral dev board). For me this works well although I need further optimization to eliminate the detection of stationary objects. 
You can further improve this system by using mqtt to automate tasks or even integrate with Blue Iris: https://community.home-assistant.io/t/person-detection-with-blue-iris-google-coral-and-frigate-v0-8-0/254747

**Comment by Shred99** [comment]
> (RS online) who say they expect a usb coral on 25 February

Unfortunately, I think RS Online's ETAs for the Corals are just broken.

I ordered one in Mid November 2021 - ETA late January 2022.  In early December, the ETA on the web site for new orders was early January, so I ordered another.  Here we are in February, with the one I ordered FIRST being due in late April and the one I ordered later, having an ETA of 25/2/2022.  Each day, I log in and check the ETAs and find that both ETA dates have increased by one day.

I think this is RSOnline's way of telling me that their actual ETA is "early 2023" like everyone else!
